main.gms

```diff
@@ -1407,6 +1407,9 @@ $setGlobal cm_import_ariadne  off !! def off
 *** then the values from the region group disaggregation will be overwritten by this region-specific value.
 *** For example: "2030.2050.MEA.EU27_regi.seh2 0.5, 2030.2050.MEA.DEU.seh2 0.3".
 $setGlobal cm_trade_SE_exog off !! def off
+*** cm_SEtaxRampUpParam "set the logistic function parameters to describe relationship between SE electricity tax rate and share of technology in total electricity demand"
+*** cm_SEtaxRampUpParam = "off" disables v21_tau_SE_tax 
+$setGlobal cm_SEtaxRampUpParam  GLO.elh2.a 0.4, GLO.elh2.b 10    !! def = GLO.elh2.a 0.4, GLO.elh2.b 10
```

Comment:
Let's be a bit more verbose here and add

```

*** This allows to manually adjust the ramp-up curve of the SE tax on electricity. It is mainly used for taxing electricity going into electrolysis for green hydrogen production.
*** The ramp-up curve is a logistic function that determines how fast taxes increase with increasing share of technology in total power demand.
*** This essentially makes an assumption about to what extend the power demand of electrolysis will be taxed and how much tax exemptions there will be at low shares of green hydrogen production.
*** The parameter a defines how fast the tax increases with increasing share, while the parameter b defines at which share
*** the tax is halfway between the value at 0 share and the maximum value (defined by a region's electricity tax and the electricity grid cost) that it converges to for high shares.
*** Example use: ...
*** For details, please see ./modules/21_tax/on/equations.gms.
```

Reply:
https://github.com/remindmodel/remind/pull/1668/commits/87f9027e8b8a3ea2e07c52b9d344db20ccbefe68